src/Panel/File.php

```diff
@@ -374,13 +376,19 @@ public function pickerData(array $params = []): array
 			};
 		}
 
-		$params['text'] ??= '{{ file.filename }}';
+		$item = new FileItem(
+			file: $this->model,
+			dragTextIsAbsolute: $absolute,
```

Comment:
```suggestion
			dragTextIsAbsolute: $absolute ?? false,
```

`Suggestion:` I often prefer adding those fallback values as actual fallbacks instead of adding the variable on top and having to keep track of the initial value as fallback (mentally).

Reply:
I agree, but in this case the default should be "true" in my opinion and it's more of a special here that it is false.